### PR TITLE
Add read_configuration For Mount System Backend Class

### PR DESCRIPTION
**Pull request by vamshideveloper**
Updated with read_configuration 
Need it to get full configuration response to check whether mount is configured or not

### CODE REVIEW

**Comment by jeffwecan** [comment]
Looks like the Travis ci build failed due to a small linting issue:

> flake8 run-test: commands[0] | flake8
./hvac/api/system_backend/mount.py:126:31: W291 trailing whitespace

File: hvac/api/system_backend/mount.py [hunk 1]
```diff
@@ -106,6 +106,25 @@ def disable_secrets_engine(self, path):
             url=api_path,
         )
 
+    def read_configuration(self, path):
```
Comment (jeffwecan) [inline hvac/api/system_backend/mount.py]: So this doesn't seem like a `/sys/mount` sort of method in particular so I don't know if it fits within this class's scope. I.e., while _some_ secrets engines have a `/config` route it isn't universal. Is there a particular set of secrets engines, etc. that you would use this method for? If so, I would say the method would better fit under those engines' classes.

Comment (vamshideveloper) [reply hvac/api/system_backend/mount.py]: Sure then i will add this under the Active directory(secret engine) class
I read configuration to skip overriding configuration by reading the existing config

**Comment by vamshideveloper** [comment]
Added read_configuration() to ad.py PR ( https://github.com/hvac/hvac/pull/507 )
you can close this PR
Thank you

**Pull request closed without merging**